docs/unified-test-documentation/dasharo-compatibility/316-sdcard-reader.md

```diff
@@ -142,18 +143,14 @@ can be used from the operating system.
 1. Power on the DUT.
 1. Boot into the system.
 1. Log into the system by using the proper login and password.
-1. Open `Windows Explorer` and go to `This PC`
-1. Find SD card icon and double click it
-1. Click the right mouse button and choose `New` then choose `Text Document`
-1. Name the new file and double click the icon to open it in `Notepad`
-1. Write any text and press Ctrl+s
-1. Close `Notepad`
-1. In the `Windows Explorer` use the right mouse button to click icon
-   of the SD Card and click `Eject`
-1. Remove the SD Card and insert it again
-1. Find SD card icon and double click it again
-1. Find created text file, double click to open it and note the result
+1. Open PowerShell as administrator.
+1. Run below commands and note results:
```

Comment:
I think, that we should add in this description information about how to determine which disk is an SD card. Not always the F:\ will be an SD card.

Reply:
Changes that you suggested have been made.